src/ecdsa/ecdsa.py

```diff
@@ -467,3 +467,15 @@ def point_is_valid(generator, x, y):
 generator_brainpoolp512r1 = ellipticcurve.PointJacobi(
     curve_brainpoolp512r1, _Gx, _Gy, 1, _q, generator=True
 )
+
+# Weierstass Curve 25519
+_a = 0x2aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa984914a144
+_b = 0x7b425ed097b425ed097b425ed097b425ed097b425ed097b4260b5e9c7710c864
+_p = 0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffed
+_Gx = 0x2aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaad245a
+_Gy = 0x20ae19a1b8a086b4e01edd2c7748d14c923d4d7e6d7c61b229e9c5a27eced3d9
+_r = 0x1000000000000000000000000000000014def9dea2f79cd65812631a5cf5d3ed
```

Comment:
where did you get those parameters? they don't seem to match the params from the draft I quoted

Reply:
I ported the parameters from this [issue](https://github.com/indutny/elliptic/pull/113)

Reply:
I have checked the parameter in [your link](https://tools.ietf.org/id/draft-struik-lwip-curve-representations-00.html) I think it might be the same.

![image](https://user-images.githubusercontent.com/5810073/89855482-c267ae80-dbc0-11ea-96e0-baa0fb66e18f.png)
![image](https://user-images.githubusercontent.com/5810073/89854982-84b65600-dbbf-11ea-8cd7-cdc6f1e483cc.png)
![image](https://user-images.githubusercontent.com/5810073/89855399-93e9d380-dbc0-11ea-83f0-9c25581ee484.png)

where _a, _b and _Gx, _Gy map to a, b and Gx', Gy' *Weierstrass curve-specific parameters* consequently and _p and _r map to p and n *General parameters* consequently.

Reply:
ah, silly me, I was trying to match the n1, which is the order of the twisted version, yeah, they look fine